src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ Notes](../../RELEASENOTES.md).
 * Optimize performance of `TraceContextPropagator.Extract`.
   ([#5749](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5749))
 
+* Remove .NET 7 TargetFramework which reached End of Support
```

Comment:
Shall I added an entry in each package's CHANGELOG.md file?

Reply:
we don't target `net7.0` for api and sdk.